tools/dht_put.cpp

```diff
@@ -122,10 +123,105 @@ void put_string(entry& e, boost::array<char, 64>& sig, boost::uint64_t& seq
 		, sig.data());
 }
 
-void bootstrap(lt::session& s)
+void bootstrap(session& s)
 {
 	printf("bootstrapping\n");
 	wait_for_alert(s, dht_bootstrap_alert::alert_type);
+	printf("bootstrap done.\n");
+}
+
+int dump_key(char *filename)
+{
+	FILE* f = fopen(filename, "rb+");
+	if (f == NULL)
+	{
+		fprintf(stderr, "failed to open file \"%s\": (%d) %s\n"
+			, filename, errno, strerror(errno));
+		return 1;
+	}
+
+	unsigned char seed[32];
+	fread(seed, 1, 32, f);
```

Comment:
would you mind checking the return value here? (make sure it's 32, and fail if it isn't)